### DISCUSSION TITLE
Issue with Button

### DISCUSSION THREAD

**Discussion by xobjects** [swordev/suid#101] [Q&A] [answered]
Every place that I use <Button> I get a an error though everything otherwise seems to work fine.

The error is ButtonBase.jsx:267 MUI: The `component` prop provided to ButtonBase is invalid.
Please make sure the children prop is rendered in this custom component.

I have tried numerous variations of using Button, but this always appears in console window.  Any thoughts?
I hate to simply ignore it, but it is otherwise working just fine.

**Comment by juanrgm** [comment, marked as answer]
This issue was reported [here ](https://github.com/swordev/suid/issues/92)and fixed in [@suid/material:0.6.1](https://github.com/swordev/suid/releases/tag/%40suid%2Fmaterial%400.6.1).
